paramak/parametric_components/blanket_cutter_parallels.py

```diff
@@ -0,0 +1,118 @@
+
+from collections import Iterable
+
+import cadquery as cq
+from paramak import ExtrudeStraightShape
+
+
+class BlanketCutterParallels(ExtrudeStraightShape):
+    """Creates an extruded shape with a parallel rectangular section repeated
+    around the reactor. The shape is used to cut other components (eg. blankets
+    and firstwalls) in order to create a banana section of the blankets with
+    parrallel sides.Typically used to divide a blanket into vertical
+    sections with a fixed distance between each section.
+
+    Args:
+        distance (float): extruded distance (cm) of the cutter which translates
+            to being the gap size between blankets when the cutter is used to
+            segment blankets.
+        gap_size (float): the distance between the inner edges of the two
+            parrallel extrusions
+        azimuth_placement_angle (float, optional): the azimuth angle(s) used
```

Comment:
shouldn't it be `float or list` ?

Reply:
ah yes thanks for that, I shall correct

Reply:
I've added a diagram to the PR, I think gap_size is just a float. 

I had copied azimuth_placement_angle from shape.py so I have now added iterable to that docstring in this commit 
https://github.com/ukaea/paramak/pull/399/commits/f75a1cb89a93f68f606bc6b230affb21c8b5a314

Reply:
You are right, I was talking about azimuth placement angle :-)